docs/conf.py

```diff
@@ -57,9 +57,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = '1.0.0-beta.5-SNAPSHOT'
+version = '1.0.0-beta.5'
```

Comment:
```suggestion
version = '1.0.0-beta.6'
```

Reply:
It needs to be 6-SNAPSHOT, if you update your comment I can accept your suggestion :)